bk-nodes/bk_db.py

```diff
@@ -0,0 +1,241 @@
+
+import MySQLdb
+import config
+import dateutil.parser
+
+
+table_fields = {}
+table_fields_index ={}
+
+
+class BeekeeperDB():
+    def __init__ ( self , retries=60) :
+
+        if not config.mysql_host:
```

Comment:
do you want to additionally check for `config.mysql_user`, `config.mysql_password` and `config.mysql_db`?

Reply:
done